PhotoPoints/app/src/main/java/edu/cascadia/mobas/photopoints/ui/map/MapFragment.java

```diff
@@ -208,3 +200,34 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
         }
     }
 }
+
```

Comment:
We should not comment old code. Instead, we should remove it. Github is our "original code" bucket :) !

Reply:
Removed in #45